pkg/operator/ceph/cluster/mon/mon_test.go

```diff
@@ -104,7 +104,7 @@ func newCluster(context *clusterd.Context, namespace string, allowMultiplePerNod
 		ClusterInfo: nil,
 		context:     context,
 		Namespace:   namespace,
-		rookVersion: "myversion",
+		rookImage:   "myversion",
```

Comment:
These changes may not be technically necessary, but I wasted at least 5 hours adding the operator config to this struct and all of the code places where it is called by parents before I realized that `rookVersion` was being used to hold the rook image string. I'd rather this not happen to anyone else.

Reply:
Agreed, I have been confused by this as well